srv/update/cmd_reify.js

```diff
@@ -0,0 +1,35 @@
+
+module.exports = function make_reify() {
+  return async function reify(msg) {
+    const seneca = this
+
+    const pkgs = await seneca.make('nodezoo', 'orig')
+      .list$({ reified_at: null, fields$: ['name'] })
```

Comment:
The reification pausing/resuming capabilities are currently in the design stage.